dotnet-logging/LogLevelsWithSerilog/LogLevelsWithSerilog/Program.cs

```diff
@@ -0,0 +1,49 @@
+using Serilog;
+using Serilog.Sinks.MSSqlServer;
+
+IConfigurationRoot configuration = new ConfigurationBuilder()
+           .SetBasePath(Directory.GetCurrentDirectory())
+           .AddJsonFile("appsettings.json")
+           .AddJsonFile("appsettings.Development.json")
+           .Build();
+
+Log.Information("Web Host started");
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddSerilog(options =>
+        {
+            //we can configure serilog from configuration
+            options.ReadFrom.Configuration(configuration);
+
+            //or we can configure serilog via fluent api
+            options.MinimumLevel.Information()
+                   .WriteTo.Console(Serilog.Events.LogEventLevel.Warning, outputTemplate: "[{Timestamp:HH:mm:ss} {Level:u3}] {Message:lj}{NewLine}{Exception}")
+                   .WriteTo.File("logs/myapplogs.txt",
+                        rollingInterval: RollingInterval.Day,
+                        rollOnFileSizeLimit: true,
+                        restrictedToMinimumLevel: Serilog.Events.LogEventLevel.Information)
+                   .WriteTo.MSSqlServer(
+                        connectionString: "Server=localhost;Database=LogDb;Integrated Security=SSPI;",
+                        sinkOptions: new MSSqlServerSinkOptions { TableName = "Logs" },
+                        restrictedToMinimumLevel: Serilog.Events.LogEventLevel.Fatal)
+                   .Enrich.WithProperty("AppName", "Serilog Demo")
+                   .Enrich.WithProperty("Environment", "Development")
+                   .Enrich.WithProperty("Author", "codemaze");
```

Comment:
We removed this from the article already and so it can also be removed here

Reply:
Although I don't share the same opinion with you on this matter I will update the code. 
- The reason why I will update is I am working for Code Maze, if this is a standard of Code Maze then I can adapt to this.
- The reason why  I don't share the same opinion with you is I think the reader of an article must not just find what the article says in the code repository but must find a reference project that gives him/her extra research topic on the article subject and beyond. If I were the reader of this article I would be curious when I encountered the Enrich usages and then start to search that topic. But as I said, this is just my opinion.

Reply:
For the CodeMaze article we want the code to be as simple as possible so that we can focus on what we are showing in the article. Since the `Enrich` pieces are not related to the article and aren't being explained in the article (as they are outside the scope of it), we don't need them here. This could potentially be another topic covered in a different article. This is the main reason to not include it here. This is from the CodeMaze author guidelines regarding implementation code:

> - code should be as simple as possible (as long as we’re still able to explain the concept)
> - for example, when you work with Entities, define a simple class: id and name - if nothing more is needed
>

Reply:
I removed that code. I respect Code Maze standards on this matter. As an author, I am obliged to implement these as well. But also as a software developer, I want to describe my opinion :)